frontend/src/metabase/app.js

```diff
@@ -6,6 +6,11 @@ import "classlist-polyfill";
 
 import "number-to-locale-string";
 
+// This is conditionally aliased in the webpack config.
+// If EE isn't enabled, it loads an empty file.
+// Should be imported before any other metabase import
+import "ee-overrides"; // eslint-disable-line import/no-unresolved, import/no-duplicates
```

Comment:
I can't find a specific case at the moment, but should we move it below `ee-plugins`? Will styles from ee-plugins work correctly if we are not doing so?

Reply:
nope, this should be imported before anything that might use `ttag` or `color`

Reply:
and the the thing is that our ee plugins depend on builtin plugins, which call `color`, that's why it's impossible to make this fix within or after `ee-plugins`